metricbeat/schema/mapstriface/mapstriface.go

```diff
@@ -128,7 +128,6 @@ func StrFromNum(key string, opts ...schema.SchemaOption) schema.Conv {
 func toStr(key string, data map[string]interface{}) (interface{}, error) {
 	emptyIface, err := common.MapStr(data).GetValue(key)
 	if err != nil {
-		fmt.Println(err)
 		return "", fmt.Errorf("Key %s not found", key)
```

Comment:
The fmt must definitively be removed. But we should add it to the error that is returned:

`return "", fmt.Errorf("Key %s not found: %s", key, err)`